src/components/ens-registration/Cover/Cover.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import { View } from 'react-native';
+import RadialGradient from 'react-native-radial-gradient';
+import Spinner from '../../Spinner';
+import { useTheme } from '@rainbow-me/context';
+import {
+  Box,
+  Cover,
+  Text,
+  useForegroundColor,
+} from '@rainbow-me/design-system';
+import { useENSProfileForm, useSelectImageMenu } from '@rainbow-me/hooks';
+import { ImgixImage } from '@rainbow-me/images';
+
+const alpha = '33';
+
+export default function CoverPhoto() {
```

Comment:
maybe is this file maybe not but I couldn't go to the place where I could choose a picture for cover, I was able to choose and image for pfp thi